lib/index.js

```diff
@@ -122,7 +122,15 @@ function convertDates(properties) {
 function formatTraits(traits) {
   var ret = {};
   each(function(value, key) {
-    var k = key.toLowerCase().replace(/\s/g, '_');
+    // Using split/join due to IE 11 failing to properly support regex in str.replace()
```

Comment:
So, @CarlosMecha, this should add support for everything on all browsers ([lookin at you IE](https://media.giphy.com/media/ZYSeEG1ffYzoQ/giphy.gif)), but it mayyyy be slow. I mean it's certainly not fast.

That said, keys should generally be tiny right? So it's probably not an issue.

Reply:
yes keys should be tiny and will rarely contain any of this stuff, most likely just spaces and underscores once in a while

thanks for checking this out!